src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -1972,4 +2077,25 @@ private bool MeetsIncludeExcludeCriteria(string filename)
             return ok;
         }
     }
+
+    /// <summary>
+    /// Get list of valid culture names for ValidateSet attribute.
+    /// </summary>
+    public class ValidateMatchStringCultureNamesGenerator : IValidateSetValuesGenerator
```

Comment:
Can this be internal?

Reply:
It must be public:
https://github.com/PowerShell/PowerShell/blob/fe40fbf6e1f3108846487dc7b830fb61867ea79f/src/System.Management.Automation/engine/Attributes.cs#L1715-L1723